src/main/java/kavitha/BagComparison/ArrayBag.java

```diff
@@ -0,0 +1,38 @@
+package kavitha.BagComparison;
+
+public class ArrayBag<T> {
+    private String[] mElements;
+    private static final int MAX_SIZE=100;
```

Comment:
No need of this constant value here as you are passing size as a constructor argument. Delete this line

Reply:
Okay